plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4275,7 +4276,7 @@ public void testFileModifiedTimeHiddenColumn()
             int col1 = (int) row.getField(1);
             Instant fileModifiedTime = ((ZonedDateTime) row.getField(2)).toInstant();
 
-            assertTrue(fileModifiedTime.toEpochMilli() > (testStartTime - 2_000));
+            assertTrue(abs(fileModifiedTime.toEpochMilli() - testStartTime) < 2_000);
```

Comment:
use `assertThan(...).isBetween`, otherwise in case of failure the message produced will be useless

Reply:
also, exercise all timestamp_precision values please

Reply:
does this column adhere to timestamp_precision at all?

Reply:
doesn't need to. isn't that worth making sure though?

Reply:
sure. done.